mmv1/templates/terraform/examples/healthcare_fhir_store_notification_config.tf.erb

```diff
@@ -12,7 +12,7 @@ resource "google_healthcare_fhir_store" "default" {
     label1 = "labelvalue1"
   }
 
-  notification_configs {
+  notification_config<% unless defined? version && version == "ga" %>s<% end %> {
```

Comment:
We only support one of these subfields, `send_full_resource`, in `notification_configs` (and at beta). Did you run this test locally successfully? I wouldn't expect it to pass.